src/lib/cms/types.ts

```diff
@@ -0,0 +1,6 @@
+import * as t from 'io-ts';
```

Comment:
We should use `zod` https://github.com/vriad/zod instead of `io-ts`. Api is very similar, so should be easy to switch out.